src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -62,4 +62,10 @@ See xref:mqtt.adoc[] for more information.
 
 The `AbstractInboundFileSynchronizer` now caches a filtered result of the `Session.list(remoteDirectory)` after slicing by the `maxFetchSize`.
 So, later synchronizations deal with the cache only by the `maxFetchSize` until the cache is exhausted.
-See xref:sftp/max-fetch.adoc[] for more information.
\ No newline at end of file
+See xref:sftp/max-fetch.adoc[] for more information.
+
+[[x7.0-null-safety]]
+=== Null Safety in  Spring Integration
+The Spring Integration team and community contributors have updated the codebase to use JSpecify and NullAway, implementing a comprehensive null safety implementation that uses `@NullMarked` annotations to default all types to non-null at the package level and `@Nullable` annotations to explicitly mark types that can be null.
```

Comment:
No need to mention team and community, and no need to say `Spring Integration`.
Even in the title.
This one sentence is enough.
The rest of the text replace with a link to the target chapter as we do in other sections of this what's new doc.